addons/kotsadm/alpha/tmpl-kotsadm-proxy.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: kotsadm
+spec:
+  template:
+    spec:
+      containers:
+      - name: kotsadm
+        env:
+        - name: PROXY_ADDRESS
```

Comment:
I'd planned for this to be `HTTP_PROXY` but that broke in-cluster networking so we'll have to update kotsadm to manually lookup the proxy address from this env var and construct a client for outbound requests.